### PR TITLE
Checklist UI modifications for app creation

### PR DESCRIPTION
**Pull request by milindk8**
# Description

Based on modules integration available, eg. for non-gitops in check list api GITOPS configuration is not required.
So, with API changes UI changes are handled

Fixes - https://github.com/devtron-labs/devtron/issues/1802

# Motivation

Making gitops configuration optional for user

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

* [x] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [x] I have performed a self-review of my own code
* [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/components/checkList/AppCheckModal.tsx [hunk 1]
```diff
@@ -47,12 +49,32 @@ export class AppCheckListModal extends Component<AppCheckListModalProps, AppChec
...
             return <>
                 <AllCheckModal />
             </>
         }
+        else if (this.props.appStageCompleted >= 5 && this.props.chartStageCompleted < 3){
+
+            return<>
+            <div className="">
+                <img src={Checklist} className="applist__checklist-img" />
+                <div className="cn-9 fw-6 fs-16 mt-16 mb-4">Get started!</div>
```
Comment (shivani170) [inline src/components/checkList/AppCheckModal.tsx:63]: Can you create a function for the above-repeated code and use that instead

**Review by shivani170 — APPROVED** [review]

File: src/components/checkList/ChartCheckModal.tsx [hunk 2]
```diff
@@ -77,18 +79,31 @@ export class ChartCheckListModal extends Component<ChartCheckListModalProps, Cha
...
         }
-        else if (this.state.appStageCompleted >= 6 && this.state.chartStageCompleted >= 3) {
+        else if (this.state.appStageCompleted >= 5 && this.state.chartStageCompleted >= 3) {
             //(app + chart) complete
             return <AllCheckModal />
         }
+        else if (this.state.appStageCompleted >= 5 && this.state.chartStageCompleted < 3){
+            return <div>
+                <img src={Checklist} className="applist__checklist-img" />
+                <div className="cn-9 fw-6 fs-16 mt-16 mb-4">Get started!</div>
```
Comment (shivani170) [inline src/components/checkList/ChartCheckModal.tsx:97]: same here

File: src/components/checkList/GlobalConfigCheckList.tsx [hunk 3]
```diff
@@ -47,10 +49,25 @@ export class GlobalConfigCheckList extends Component<GlobalConfigCheckListProps,
...
         }
-        else if (this.props.appStageCompleted >= 6 && this.props.chartStageCompleted >= 3) {
+        else if (this.props.appStageCompleted >= 5 && this.props.chartStageCompleted >= 3) {
             //(app + chart) complete
             return <GlobalAllCheckModal />
         }
+        else if (this.props.appStageCompleted >= 5 && this.props.chartStageCompleted < 3){
+
+            return <div className="">
+                <div className="cn-9 fw-6 fs-16 mb-8">Get started!</div>
```
Comment (shivani170) [inline src/components/checkList/GlobalConfigCheckList.tsx:59]: Same here

File: src/components/checkList/AppCheckModal.tsx [hunk 4]
```diff
@@ -47,12 +49,32 @@ export class AppCheckListModal extends Component<AppCheckListModalProps, AppChec
                     toggleChartChecklist={this.toggleChartChecklist} />
             </div>
         }
-        else if (this.props.appStageCompleted >= 6 && this.props.chartStageCompleted >= 3) {
+        else if (this.props.appStageCompleted >= 5 && this.props.chartStageCompleted >= 3) {
             //(app + chart) complete
             return <>
```
Comment (shivani170) [inline src/components/checkList/AppCheckModal.tsx:54]: No need of diamond tag here

File: src/components/checkList/AppCheckModal.tsx (hunk 1 above)
Comment (shivani170) [reply src/components/checkList/AppCheckModal.tsx:63]: No need of the diamond tag here. Can remove

**Review by vivek-devtron — APPROVED** [review]

File: src/components/checkList/AppCheckModal.tsx (hunk 4 above)
Comment (milindk8) [reply src/components/checkList/AppCheckModal.tsx]: done

**Review by sohel-rp — APPROVED** [review]

**Review by sohel-rp — APPROVED** [review]

**Pull request merged**